src/app/test-record/test-section/edit/test-section-edit.component.html

```diff
@@ -36,7 +36,215 @@
+                formGroupName="reasonForAbandoning"
+              >
+                <input
+                  class="govuk-checkboxes__input"
+                  id="test-reason-{{ i }}"
+                  type="checkbox"
+                  [formControl]="reason"
+                  [value]="this.reasonsForAbandoningOptions[i].selected"
+                />
+                <label class="govuk-label govuk-checkboxes__label">
+                  <span>{{ this.reasonsForAbandoningOptions[i].name }}</span>
+                </label>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="govuk-grid-row govuk-!-margin-top-7">
+          <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-form-group">
+              <label
+                class="govuk-label govuk-!-font-weight-bold"
+                for="additionalCommentsForAbandon"
+              >
+                Additional details for abandoning
+              </label>
+              <span id="additionalCommentsForAbandon" class="govuk-hint">
+                Notes about the reason for abandoning should be displayed here...
+              </span>
+              <textarea
+                class="govuk-textarea"
+                id="more-detail"
+                formControlName="additionalCommentsForAbandon"
+                rows="5"
+                aria-describedby="additionalCommentsForAbandon"
+                >{{ testType.additionalCommentsForAbandon }}</textarea
+              >
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div> -->
+
+  <div
+    *ngIf="testTypesApplicable.certificateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-7"
+  >
+    <div class="govuk-grid-column-one-half">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Certificate number
+      </label>
+      <input
+        class="govuk-input"
+        id="test-certificateNumber"
+        formControlName="certificateNumber"
+        type="text"
+      />
+    </div>
+  </div>
+
+  <div
+    *ngIf="testTypesApplicable.expiryDateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-3"
+  >
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Expiry Date
+      </label>
+      <vtm-date-input
+        id="test-testExpiryDate"
+        ariaDescribedBy="testExpiryDate"
+        formControlName="testExpiryDate"
+        ngDefaultControl
+      >
+      </vtm-date-input>
+    </div>
+  </div>
+
+  <div
+    *ngIf="testTypesApplicable.anniversaryDateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-3"
+  >
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Anniversary Date
+      </label>
+      <vtm-date-input
+        id="test-testAnniversaryDate"
```

Comment:
rename edit with test-edit.
We use test- for view screens and test-edit- for edit screens